Code/GraphMol/molopstest.cpp

```diff
@@ -418,7 +413,13 @@ void test4() {
   TEST_ASSERT(adjMat[1] == 1);
   TEST_ASSERT(adjMat[2] == 1);
   TEST_ASSERT(adjMat[3] == 0);
-  delete m;
+  bool useBO = true;
+  adjMat = MolOps::getAdjacencyMatrix(*m, useBO);
```

Comment:
Might want to check the prop names are being set properly here between useBO and atom Weights.

Reply:
`getAdjacencyMatrix()` doesn't have an option with atom weights.

Reply:
Wrong function, I apparently meant:  getDistanceMat

Reply:
I believe that test does that already